types/wlr_data_device.c

```diff
@@ -907,6 +907,9 @@ static void data_source_offer(struct wl_client *client,
 		*p = strdup(mime_type);
 	}
 	if (!p || !*p){
+		if (p) {
+			source->mime_types.size -= sizeof *p;
```

Comment:
`sizeof(*p)`

Reply:
I've sticked to the style of the file, 5 lines above. I guess I'll change that one as well then.